src/string_with_vars.ml

```diff
@@ -8,8 +8,8 @@ type item =
 
 type t =
   { items : item list
-  ; loc   : Loc.t
-  }
+  ; loc : Loc.t
+  ; unquoted : bool }
```

Comment:
Why not use `quoted`? I tend to think that it's more natural to store the information rather than it's negation

Reply:
It is just because the tests then repeatedly will do `not t.quoted` — but maybe this is optimized away?

Reply:
yh it probably is. I'm pretty sure that at the assembly level `not x` and `x` are exactly the same number of instructions

Reply:
OK, done.